### ISSUE TITLE
Domain activation is Failed - CNAMEAlreadyExist error but nothing exists

### ISSUE THREAD

**Issue by QB3L** [aws-amplify/amplify-console#131]
**App Id**: d31z6fkwcjb8er
**Region**: us-east-1
**Step**: Domain activation
**Status**: Failed

>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.

**Issue/question**
Domain is not activating with amplify app

**Error message**
Error message: One or more of the CNAMEs you provided are already associated with a different resource

**Additional information**
I verified that the subdomain I am registering is not being used in any cloudfront distribution (it used to be and more than 6hrs have passed) and I also made sure there wheren’t any DNS records already existing on Route53 for that subdomain

**Comment by nimacks** [comment]
@QB3L, our records indicate that your domain is enabled. I can confirm that I was able to navigate successfully to your domain. Please re-open this ticket if you're still having issues.

**Comment by QB3L** [comment]
@nimacks I ended up creating a completely new subdomain `store.mywebsite.com` to make the app deploy successfully. 
I then tried creating a route53 CNAME record to point the original subdomain  I wanted (`apps.mywebsite.com`) to the now working subdomain (`store.mywebsite.com`) but this doesn't work!

So only `store.mywebsite.com` works but not `apps.mywebsite.com`

Any help on why the apps subdomain could be failing is appreciated.
Thanks
